app/views/courses/preview/financial_support/_scholarship_and_bursary.html.erb

```diff
@@ -1,26 +1,34 @@
-<p class="govuk-body">
-  You could be eligible for either:
-</p>
-
-<ul class="govuk-list govuk-list--bullet">
-  <li data-qa="course__scholarship_amount">a scholarship of <%= number_to_currency(course.scholarship_amount) %></li>
-  <li data-qa="course__bursary_amount">a bursary of <%= number_to_currency(course.bursary_amount) %></li>
-</ul>
-
-<% if course.has_early_career_payments? %>
+<% if course.has_financial_incentives? %>
   <p class="govuk-body">
-    With a scholarship or bursary, you’ll also get early career payments of £5,000 each in your third and fifth year of teaching (£7,500 in <%= link_to 'some areas of England', 'https://www.gov.uk/guidance/mathematics-early-career-payments-guidance-for-teachers-and-schools', class: "govuk-link" %>).
+    You could be eligible for either:
```

Comment:
https://find-postgraduate-teacher-training.education.gov.uk/course/2AT/2T89#section-financial-support

https://github.com/DFE-Digital/search-and-compare-ui/blob/master/shared/Views/Shared/Components/CourseDetails/FinancialSupport/_Bursary.cshtml

![image](https://user-images.githubusercontent.com/470137/69142928-c21e0500-0abf-11ea-9004-8d9088cdb080.png)

^^ should we still be presenting like the above?